src/components/apicall.js

```diff
@@ -1,14 +1,33 @@
-const category = 'anger';
+import { useState, useEffect } from 'react';
+import './apicall.css';
 
-const getdata = async () => {
-  const response = await fetch(`https://api.api-ninjas.com/v1/quotes?category=${category}`, {
-    headers: {
-      'X-Api-Key': 'Fw08AILtO+Qt2bR7wENuqQ==K7dKDmbDUGlMhbEh',
-      'Content-Type': 'application/json',
-    },
-  });
-  const data = await response.json();
-  return data;
+const ApiNinja = () => {
+  const category = 'anger';
+  const [data, setData] = useState({ quote: 'LOADING', author: '' });
```

Comment:
- [ ] Great job working on this milestone. App works and displays quotes on refresh. As per project requirements, you should have an `error state`. When I check, that is currently missing. Kindly add this state and you will be ready to move to the next milestone.
![image](https://github.com/EmiLoKiJu/my-app/assets/62849872/8f3da504-4824-462c-8ab8-8f13fc545acf)